pkgs/servers/audiobookshelf/wrapper.nix

```diff
@@ -44,7 +44,7 @@
           ;;
         --help|-h)
           echo "Usage: audiobookshelf [--host <host>] [--port <port>] [--metadata <dir>] [--config <dir>]"
-          shift
+          exit 0
```

Comment:
@jvanbruegge Are you also ok with this change? I didn't check the PR state before pushing. Currently --help prints help and proceeds to start that application anyway.

Reply:
Yeah that was probably a copy paste mistake

Reply:
@jvanbruegge Please re-approve.